manifests/osfamily/suse.pp

```diff
@@ -16,9 +16,9 @@
     }
     '11', '12': {
       # Import the GPG key
-      $legacy_keyname  = 'RPM-GPG-KEY-puppetlabs'
```

Comment:
Can you also update legacy_gpg_path and gpg_path to re-add the `RPM-` prefix? I did that for RedHat to avoid creating multiple files on machines that already have the key present.